foonathan-memory 0.7.4

jeongseok-meta opened the pull request:
Closes #11 

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

jeongseok-meta committed:
foonathan-memory 0.7.4

jeongseok-meta committed:
foonathan-memory 0.7.4

jeongseok-meta commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14807694905](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14807694905). Examine the logs at this URL for more detail.</sub>

jeongseok-meta committed:
Fix foonathan-memory 0.7.4 build

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

jeongseok-meta committed:
Fix doctest availability on linux variants

jeongseok-meta committed:
MNT: Re-rendered with conda-smithy 3.61.2 and conda-forge-pinning 2026.05.09.13.09.17

Other tools:
- conda-build 26.3.0
- rattler-build 0.64.1
- rattler-build-conda-compat 1.4.14

jeongseok-meta commented:
CI is green on the current head and this is ready for review. Local validation also covered recipe lint, a native linux-64 conda build with upstream ctest, the downstream CMake package consumer test, and linux-aarch64/linux-ppc64le solve checks.

Tobias-Fischer commented:
Thanks a lot @jeongseok-meta!
